spec/api-menu-spec.js

```diff
@@ -283,7 +283,52 @@ describe('Menu module', () => {
     })
 
     it('returns immediately', () => {
-      menu.popup(w, {x: 100, y: 100, async: true})
+      const { browserWindow, x, y } = menu.popup(w, {x: 100, y: 101})
+
+      assert.equal(browserWindow, w)
+      assert.equal(x, 100)
+      assert.equal(y, 101)
+
+      menu.closePopup(w)
```

Comment:
If the test fails the created popup won't be closed.
Was it the reason why the `afterEach` hook was timing out on Mac?